### PR TITLE
blog: adding 2 new blogs for gravel and general update

### PR DESCRIPTION
**Pull request by asettle**
Signed-off-by: Alexandra Settle <asettle@suse.com>

### CODE REVIEW

File: _posts/2021-05-28-state-of-the-union.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- We now have a [feature-based development roadmap](https://github.com/aquarist-labs/aquarium/blob/main/doc/project-plan/roadmap.rst)
+- We refactored the installer: Now it uses the wizard style in both
+  scenarios (setting up new node and adding one to an existing cluster)
+- We have 48 GitHub stars and over 67,000 lines of code
+- Our website has had a facelift and we now store our meeting minutes
+  openly.
+
+Overall, we're in a really good place for Aquarium. Glass is based on Google's
+Angular & Angular Material framework. The entire code is written in TypeScript
+which is transpired into JavaScript to be able to run on various browsers.
```
Comment (jecluis) [inline _posts/2021-05-28-state-of-the-union.md:24]: I suspect that, where it reads `transpired`, it should read `transpiled`.

**Review by jecluis — DISMISSED** [review]
LGTM, but I suspect there's a typo.

File: _posts/2021-05-28-state-of-the-union.md (hunk 1 above)
Comment (asettle) [reply _posts/2021-05-28-state-of-the-union.md:24]: Good catch, cheers.

**Review by jecluis — APPROVED** [review]

**Review by l-mb — APPROVED** [review]
LGTM. Thanks for writing this up!

**Pull request merged**